pkg/rfc8693/claims.go

```diff
@@ -44,37 +46,56 @@ func parseCEL(input string) (cel.Program, error) {
 	return prog, nil
 }
 
+type subToCEL map[string]cel.Program
```

Comment:
Is this a more general type, and if so would a name like `claimToCEL` make more sense?